src/app/debug/table/table.component.css

```
@@ -79,19 +78,14 @@ th {
 
 .progress-report-timout {
```

```suggestion
.progress-report-timeout {
```
Small typo. See below for reference in table template

---

fixed